include/sys/vdev_disk.h

```diff
@@ -37,11 +37,5 @@ typedef struct vdev_disk {
 	struct block_device	*vd_bdev;
 } vdev_disk_t;
 
-#ifndef __linux__
-extern int vdev_disk_physio(struct block_device *, caddr_t,
```

Comment:
Now __vdev_disk_physio() can be simplified, it now only takes zio as argument since vdev_disk_physio() has been removed. Also, I think __vdev_disk_physio() can be renamed as vdev_disk_physio().

I'll clean up the code in a patch I'm working on to make vdev_disk.c use ABD pages.